framework/api/infrastructure/middleware.py

```diff
@@ -27,16 +28,42 @@ async def deserialise_web_request():
         _RequestData: dict = request.get_json()
 
         for _AttributeName, _AttributeType in get_type_hints(REQUEST_OBJECTS_BY_ENDPOINT[_RequestEndpoint]).items():
-            _Data = _RequestData[_AttributeName]
+            
+            _Data = _RequestData.get(_AttributeName)
+
+            if isinstance(_AttributeType(_Data), AttributeChangeTracker):
+                __deserialise_attribute_change_tracker(_AttributeName, _RequestData)
+                continue
 
             if _AttributeType is bytes:
                 _Data = b64decode(_Data)
 
             _RequestData[_AttributeName] = _AttributeType(_Data) if _Data else None
-
+        
         _DeserialisedRequest = REQUEST_OBJECTS_BY_ENDPOINT[_RequestEndpoint](**_RequestData)
         setattr(request, "request_object", _DeserialisedRequest)
 
+def __deserialise_attribute_change_tracker(attribute_name: str, request_data: dict):
+    '''
+
+        Args:
+            attribute_name (str): The name of the attribute 
+            request_data (dict): The request data which may require deserialisation for the value
+                of key 'attribute_name'. If not found
+                The deserialised attribute change tracker will be inserted or override the corresponding 
+                value for 'attribute_name'... STILL WRITING DOCUMENTATION
+    '''
+    
+    if attribute_name not in request_data.keys():
```

Comment:
would this read better as if, elif & else without the returns?

Reply:
Reads better & more succinct. Next commit will contain this change.